src/pyscipopt/scip.pyx

```diff
@@ -474,15 +474,38 @@ cdef class NLRow:
 cdef class Solution:
     """Base class holding a pointer to corresponding SCIP_SOL"""
     cdef SCIP_SOL* sol
+    cdef SCIP* scip
     # can be used to store problem data
     cdef public object data
 
     @staticmethod
-    cdef create(SCIP_SOL* scip_sol):
+    cdef create(SCIP* scip, SCIP_SOL* scip_sol):
         sol = Solution()
         sol.sol = scip_sol
+        sol.scip = scip
         return sol
 
+    def __getitem__(self, Variable var):
+        return SCIPgetSolVal(self.scip, self.sol, var.scip_var)
+
+    def __setitem__(self, Variable var, value):
+        PY_SCIP_CALL(SCIPsetSolVal(self.scip, self.sol, var.scip_var, value))
+
+    def __repr__(self):
+        cdef SCIP_VAR* scip_var
+
+        vals = {}
+        for i in range(SCIPgetNVars(self.scip)):
+            scip_var = SCIPgetVars(self.scip)[i]
+
+            # exctract name
+            print(scip_var == NULL)
```

Comment:
Do we need this `print`?

Reply:
Oh, sorry. I was just debugging a bit and accidentally added it.

Reply:
Fixed with ee2df06.